### ISSUE TITLE
New releases?

### ISSUE THREAD

**Issue by dpisano** [adammck/terraform-inventory#99]
Would it be possible to make a new releases?

**Comment by aisrael** [comment]
Yeah the `brew install terraform-inventory` still picks up an older release (that doesn't work with Terraform v0.11.11).

**Comment by adammck** [comment]
Just released [v0.8](https://github.com/adammck/terraform-inventory/releases/tag/v0.8). Sorry about the delay; this project is a bit neglected these days, but only by me, because this release contains a lot of great new features and fixes. Thanks, everyone!

**Comment by ballad89** [comment]
why is this still installing 0.6.1 ?
```bash
11:21 $ brew install terraform-inventory
==> Downloading https://homebrew.bintray.com/bottles/terraform-inventory-0.6.1.mojave.bottle.tar.gz
Already downloaded: /Users/husaynarrah/Library/Caches/Homebrew/downloads/17f642cf3cf0fd9d67fc10b03b269ab8d971f9d13dda624c5fdebadcc2ae8e75--terraform-inventory-0.6.1.mojave.bottle.tar.gz
==> Pouring terraform-inventory-0.6.1.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/terraform-inventory/0.6.1: 4 files, 2.8MB
```
